docs/en/ingest-management/security/certificates.asciidoc

```diff
@@ -0,0 +1,256 @@
+tab in {fleet} and follow the instructions to generate the service token now.
++
+TIP: The quick start command shown in the instructions is incomplete. You need
+to add the settings shown in the next step before running the command. 
+ 
+.. From the directory where the {fleet-server} agent is installed, run the
+`enroll` (or `install`) command and specify the certificates to use. For example:
++
+--
+[source,shell]
+----
+sudo ./elastic-agent enroll -f \
+   --url=https://192.0.2.1:8220 \
+   --fleet-server-es=https://192.0.2.0:9200 \
+   --fleet-server-service-token=AAEBAWVsYXm0aWMvZmxlZXQtc2XydmVyL3Rva2VuLTE2MjM4OTAztDU1OTQ6dllfVW1mYnFTVjJwTC2ZQ0EtVnVZQQ \
+   --fleet-server-es-ca=/path/to/elasticsearch-ca.crt \
+   --certificate-authorities=/path/to/ca.crt \
+   --fleet-server-cert=/path/to/fleet-server.crt \
+   --fleet-server-cert-key=/path/to/fleet-server.key
+----
+--
++
+Where:
++
+--
+`url`::
+{fleet-server} URL
+`fleet-server-es`::
+{es} URL
+`fleet-server-service-token`::
+Service token to use to communicate with {es}
+`fleet-server-es-ca`::
+CA certificate to use to connect to {es}
+`certificate-authorities`::
+CA certificate to use to connect to {fleet-server}
+`fleet-server-cert`::
+Certificate to use for the exposed {fleet-server} HTTPS endpoint
+`fleet-server-cert-key`::
+Private key to use for the exposed {fleet-server} HTTPS endpoint
+--
++
+.What happens if you enroll {fleet-server} without specifying certificates?
+****
+
+If the certificates are managed by your organization and installed at the system
+level, they will be used to encrypt traffic between {agent}s, {fleet-server},
+and {es}.
+
+If system-level certificates don't exist, {fleet-server} automatically generates
+self-signed certificates. Traffic between {fleet-server} and {agent}s over
+HTTPS is encrypted, but the certificate chain cannot be verified. Any {agent}s
+enrolling in {fleet-server} will need to pass the `--insecure` flag to
+acknowledge that the certificate chain is not verified.
+
+Allowing {fleet-server} to generate self-signed certificates is useful to get
+things running for development, but not recommended in a production environment.
+****
+
+.. Start the {fleet-server} agent.
+
+. Next, enroll (or install) your {agent}s into {fleet}. {agent}s connecting to a
+secured {fleet-server} need to pass in the CA certificate used to generate the certificate used by the
+{fleet-server}. The CA certificate used by {es} is already specified in the
+agent policy because it's set under {fleet} settings in {kib} and does not need to be passed in on the command line.
++
+[source,shell]
+----
+sudo elastic-agent enroll -f --url=https://192.0.2.1:8220 \
+  --enrollment-token=<< enrollment token for the policy that will be applied >> \
+  --certificate-authorities=/path/to/ca.crt
+----
+
+//TODO: I need to decide how to handle enroll vs install command because it
+//depends on OS and whether the user has already installed Fleet Server before
```

Comment:
Install command works on all OS's.

Reply:
@blakerouse I used the wrong word (OS) here....if you install using DEB or RPM, you can't use the install command, correct?

Reply:
I've added some clarification and made install the main step.